include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -1091,6 +1091,24 @@ def FIntegerConstantOp : FIRRTLOp<"integer", [Pure, ConstantLike]> {
   let hasCustomAssemblyFormat = true;
 }
 
+def PathOp : FIRRTLOp<"path", [Pure,
+                  DeclareOpInterfaceMethods<InnerRefUserOpInterface>]> {
+  let summary = "Produce a path value";
+  let description = [{
+    Produces a value which represents the path to a hardware entitiy in a
```

Comment:
Well we can tweak, but is this the path /to/ an entity of the path of an entity? I think it'll be plumbed around and thereby describe the path of the target, kinda like refs (?), but not sure.

Reply:
```suggestion
    Produces a value which represents the path to a hardware entity in a
```